tests/notification.json

```diff
@@ -0,0 +1 @@
+{"slack": {"webhook": "123456"},"ipython" : {"webhook": "1234"}}
```

Comment:
is it needed?

Reply:
i added this file to test setting notifications using a JSON file, so i think we need to keep it